source/core/NumericString.cpp

```diff
@@ -1868,7 +1829,31 @@ VIREO_FUNCTION_SIGNATUREV(StringFormat, StringFormatStruct)
         format.AliasAssign(tempformat.Begin(), tempformat.End());
     }
     StringRef buffer = _Param(StringOut);
-    Format(&format, count, arguments, buffer);
+    Format(&format, count, arguments, buffer, null);
+    return _NextInstruction();
+}
+struct StringFormatWithErrStruct : public VarArgInstruction
+{
+    _ParamDef(StringRef, StringOut);
+    _ParamDef(StringRef, StringFormat);
+    _ParamDef(ErrorCluster, ErrClust);
+    _ParamImmediateDef(StaticTypeAndData, argument1[1]);
+    NEXT_INSTRUCTION_METHODV()
+};
+
+VIREO_FUNCTION_SIGNATUREV(StringFormatWithErr, StringFormatWithErrStruct) {
```

Comment:
This code is hard to share because both functions take structure arguments, with a difference in an interior field.  If it were refactored, most of the code would still be pulling fields out of the structure to pass to a small common function.  Since the non-error I/O version of this is expected to be removed as soon as DFIR is refactored not to generate it, I think this is not worth doing.